test/test_restful.py

```diff
@@ -828,6 +828,17 @@ def test_034_cv_agent_post_invalid_exclude_list(self):
         json_response = response.json()
         self.assertIn("results", json_response, "Malformed response body!")
 
+    def test_035_test_delete_in_use_allowlist(self):
```

Comment:
This is a good test of this issue, but looking at the other tests, I don't see a test that successfully deletes an allowlist. `test_027_cv_allowlist_delete` and `test_035_test_delete_in_use_allowlist` both fail in different ways. Do we need another one to make sure deletes still work?

Reply:
@mpeters Doesn't `test_027_cv_allowlist_delete` pass if successful? The error message is only returned if the status _doesn't_ match `204`, which is success.

Reply:
You're right, sorry I was reading the assert message and got my logic confused.